magstock/templates/regextra.html

```diff
@@ -60,14 +60,18 @@
 </script>
```

Comment:
This section of if statements are starting to get pretty hairy, at some future point we should simplify it.

I tested the hell out of this with the food cap not hit, with it hit, and with combinations of guest badges, staff badges, attendee badges in band groups, etc, and it seemed to work just fine.